redesign/about.html

```diff
@@ -0,0 +1,183 @@
+        <i class="fa fa-bars white dib f2"></i>
+      </label>
+      <ul class="overflow-hidden menu db-ns w-100 list tc pl0 pt3 mv0 f3 fw3 f5-ns">
+        <li class="absolute top-1 static-ns ph4 mh2 fw3 di-ns pt1 pb3 pv3-ns">
+          <a href="/" class="white link">
+            <img src="./img/common/dwyl-heart-only-logo.png" alt="dwyl logo" class="w1-plus"/>
+            dwyl
+          </a>
+        </li>
+        <li class="ph5 pt5 pb2 tl dn-ns"><a href="/" class="white link">Home</a></li>
+        <li class="ph5 di-ns pv2 tl pv0-ns"><a href="/portfolio" class="white link">Portfolio</a></li>
+        <li class="ph5 di-ns pv2 tl pv0-ns"><a href="/values" class="white link">Values</a></li>
+        <li class="ph5 di-ns pv2 tl pv0-ns"><a href="/team" class="white link">Team</a></li>
+        <li class="ph5 di-ns pv2 tl pv0-ns"><a href="/app" class="white link">App
+          <span class="dwyl-bg-gold pa1 br2 f7">BETA</span></a></li>
+        <li class="ph5 di-ns pv2 tl pv0-ns"><a href="/blog" class="dwyl-mint link">Blog</a></li>
+        <li class="ph5 di-ns pt2 pb3 tl pv0-ns"><a href="/#contact" class="white link">Contact</a></li>
+      </ul>
+    </nav>
+
+    <section class="dwyl-dark-gray pt6 pb3 pb3-ns ph6-ns center shadow-4 tc">
+      <img src="./img/common/dwyl-heart-only-logo.png" alt="dwyl logo" class="w4 tc"/>
+      <h1 class="f1">Driven by Our Values</h1>
+    </section>
+
+    <section class="dwyl-dark-gray pt3 pb3 pb5-ns ph6-ns center tl w-60-l w-90">
+      <h2 class="dwyl-mint bb dwyl-b--gray pl3">dwyl</h2>
+      <p class="pl3 b mv1 f6">NOUN</p>
+      <div>
+        <p class="pl3 b dib mv1">Pronounciation:</p>
+        <p class="dib pl3 mv1 times-new-roman">/dwɪɫ/</p>
+      </div>
+      <div>
+        <p class="pl3 b dib mv1">Origin:</p>
+        <p class="dib pl3 i mv0 times-new-roman">Do What You Love</p>
+      </div>
+      <div>
+        <p class="pl3 b dib mv1 dib">1.</p>
+        <p class="dib pl2 i mv0 times-new-roman w-80 w-90-ns v-top pt1">
+          Community of people on a mission to change the world using technology.
+        </p>
+      </div>
+
+    </section>
+
+    <section class="dwyl-bg-dark-gray white pt5 pb4 pb5-ns ph6-ns ph4 center">
+      <h2 class="f2 mv0">More Than Just Software</h2>
+      <p class="istok-web">We solve real problems with technology. We are different by design.</p>
+      <button class="bn dib link mb3 ph3 pv2 dwyl-bg-yellow dwyl-dark-gray br2 f5 shadow-4 pointer istok-web">Join Our Community</button>
+      <button class="bn dib link ml2 mb3 ph3 pv2 dwyl-bg-mint white br2 f5 shadow-4 pointer istok-web">Meet the Team</button>
+
+      <h2 class="f2 mt5 mb2">We believe in:</h2>
+      <div class="dib w-45-ns">
+        <h3 class="dwyl-yellow mt1">People 🤝</h3>
+        <p class="lh-copy">
+          Our people are our primary focus, not profit. We have a
+          <a href="#" class="dwyl-yellow">worldwide community of over 350 creative technologists.</a>
```

Comment:
Not sure where this should link to. We have 400+ people now on github, but 'only' 134 have publicised their membership of the org on GH, so if you're _not_ a member of dwyl already, you only see 134 when you navigate to: https://github.com/orgs/dwyl/people

Reply:
Not sure either... was following the designs. @markwilliamfirth any thoughts? Otherwise I'm happy to just remove this link but keep the text.

Reply:
Doesn't it link to the community page?

Reply:
I'll make it go there, thanks :blush: